examples/protobuf_producer.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# This is a simple example of the SerializingProducer using protobuf.
+#
+# To regenerate Protobuf classes you must first install the protobuf
```

Comment:
Do they have to? I thought we checked in the generated classes?

Reply:
That's why it says regenerate as opposed to generate. The implication being if you want to generate them again. I can remove it or say if you want to make changed to the proto definition you need to regenerate them and you can use make.

Reply:
Ah, sorry, read that as "generate". It is fine as is-is then :+1: